src/hub.js

```diff
@@ -462,20 +463,21 @@ document.addEventListener("DOMContentLoaded", async () => {
 
   const subscriptions = new Subscriptions(hubId);
 
-  if (navigator.serviceWorker) {
-    try {
-      navigator.serviceWorker
-        .register("/hub.service.js")
-        .then(() => {
-          navigator.serviceWorker.ready
-            .then(registration => subscriptions.setRegistration(registration))
-            .catch(() => subscriptions.setRegistrationFailed());
-        })
-        .catch(() => subscriptions.setRegistrationFailed());
-    } catch (e) {
-      subscriptions.setRegistrationFailed();
-    }
-  }
+  //if (navigator.serviceWorker) {
```

Comment:
This serviceWorker initialization seems to break entry on chrome. I don't know why. 
It seems that neither the `then` nor the `catch` attached to `navigator.serviceWorker.ready` is reached.